docs/services/Facebook.md

```diff
@@ -83,10 +83,10 @@ await FacebookService.Instance.PostToFeedWithDialogAsync(TitleText.Text, Descrip
 await FacebookService.Instance.PostToFeedAsync(TitleText.Text, MessageText.Text, DescriptionText.Text, picture.Name, stream);
 
 // Get current user's photo albums
-await FacebookService.Instance.GetUserAlbumsAsync();
+await FacebookService.Instance.GetUserAlbumsAsync(20, FacebookAlbum.Fields);
```

Comment:
Either GetUserAlbumsAsync() or GetUserAlbumsAsync(20) result in an ambiguous call:

The call is ambiguous between the following methods or properties: 'FacebookService.GetUserAlbumsAsync(int, string)' and 'FacebookService.GetUserAlbumsAsync(int, string, int)'

Here are the two overloads:

1. public async Task<List<FacebookAlbum>> GetUserAlbumsAsync(int maxRecords = 20, string fields = null)

2. public async Task<IncrementalLoadingCollection<FacebookRequestSource<FacebookAlbum>, FacebookAlbum>> GetUserAlbumsAsync(int pageSize = 20, string fields = null, int maxPages = 5)

#1 is the original, unchanged signature.  I could remove optionals off of #2 to resolve?  New signature would be:

public async Task<IncrementalLoadingCollection<FacebookRequestSource<FacebookAlbum>, FacebookAlbum>> GetUserAlbumsAsync(int pageSize, int maxPages, string fields = null)